### PR TITLE
Gio/refactor buttons

### PR DESCRIPTION
**Pull request by digmel**
Old button change by a new one, refactor,  uppercase prop added

### CODE REVIEW

File: src/components/button/Button.tsx [hunk 1]
```diff
@@ -15,11 +15,12 @@ export const Button: FC<TButtonProps> = ({
   size = EButtonSize.REGULAR,
   isFluid = false,
   isDisabled = false,
+  isUpperCase = false,
```
Comment (lasharela) [inline src/components/button/Button.tsx:18]: Mostly will be UpperCase. So, call it isLowCase and by default will be false

File: src/components/button/Button.tsx [hunk 2]
```diff
@@ -90,7 +91,9 @@ export const Button: FC<TButtonProps> = ({
           />
         )}
       </View>
-      <Text style={[styles.title, titleDynamicStyle, titleStyle]}>{title}</Text>
+      <Text style={[styles.title, titleDynamicStyle, titleStyle]}>
+        {isUpperCase ? title.toUpperCase() : title}
```
Comment (lasharela) [inline src/components/button/Button.tsx:95]: Dud. you don't really need function. It's just CSS.. there is a style for this. 
https://stackoverflow.com/questions/35813156/how-to-set-text-text-to-upper-case-in-react-native/65458073

File: src/components/button/buttonStories.tsx [hunk 3]
```diff
@@ -6,12 +6,11 @@ import {View} from 'react-native';
 
 export const buttonStories = storiesOf('Button', module);
 
-//-------Regular Button------------
-
 buttonStories.add('Regular', () => (
   <View style={{margin: 5, flex: 1, justifyContent: 'space-evenly'}}>
```
Comment (lasharela) [inline src/components/button/buttonStories.tsx:10]: We have to eliminate inline styles. In next PR we are going to create Screen component

**Review by lasharela — CHANGES_REQUESTED** [review]

File: src/components/button/Button.tsx (hunk 1 above)
Comment (digmel) [reply src/components/button/Button.tsx:18]: got it

File: src/components/button/Button.tsx (hunk 2 above)
Comment (digmel) [reply src/components/button/Button.tsx:95]: fixed

File: src/components/button/buttonStories.tsx (hunk 3 above)
Comment (digmel) [reply src/components/button/buttonStories.tsx:10]: ok

**Comment by digmel** [comment]
all comments resolved, we can merge now

**Review by lasharela — APPROVED** [review]

**Pull request merged**